lua/nvim-treesitter/incremental_selection.lua

```diff
@@ -131,15 +131,19 @@ function M.attach(bufnr)
   local config = configs.get_module "incremental_selection"
   for funcname, mapping in pairs(config.keymaps) do
     local mode
+    local rhs
     if funcname == "init_selection" then
       mode = "n"
+      rhs = M[funcname]
     else
       mode = "x"
+      -- We need to move to command mode to access marks '< (visual area start) and '> (visual area end) which are not properly accessible in visual mode.
```

Comment:
`luacheck` is complaining that this comment is too long

```suggestion
      -- We need to move to command mode to access marks '< (visual area start) and '> (visual area end) which are not
      -- properly accessible in visual mode.
```